.github/workflows/ci.yml

```diff
@@ -30,9 +30,14 @@ jobs:
           CI: true
       - name: unit test
         run: |
-          npm run test:unit
+          npm run test:unit --coverage
         env:
           CI: true
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v1
+        with:
+          name: codecov-business-filings
+          fail_ci_if_error: true
```

Comment:
Similar to Cam's comment... Should we pass CI if error? (Otherwise this might block us from integrating any code for some time while we fix things.) (Or do this on an as-needed basis only now until we catch up?)

Reply:
We can adjust the baseline target to the current coverage. Every project is different. We need a couple of PR to handle it properly.